packages/synapse-react-client/package.json

```diff
@@ -214,29 +215,36 @@
   "scripts": {
     "clean": "rimraf dist storybook-static coverage",
     "prepublishOnly": "pnpm install && pnpm build",
-    "build-gh": "pnpm build:docs",
-    "deploy": "pnpm install && pnpm build:storybook && gh-pages -t -d storybook-static -o upstream",
-    "deploy:origin": "pnpm install && pnpm build:storybook && gh-pages -t -d storybook-static -o origin",
     "start-js": "vite",
     "start": "pnpm storybook",
     "start-deprecated-demo": "start-js",
     "test": "jest",
     "test:ci": "jest --watchAll=false --coverage",
     "test:coverage": "jest --coverage",
-    "build": "pnpm build:js && pnpm build:copy-assets && pnpm build:esbuild",
-    "build:js": "tsc --build",
-    "build:copy-assets": "copyfiles -u 2 \"src/lib/**/*.{css,scss,svg}\" dist",
-    "build:esbuild": "rimraf dist/umd && node esbuild.config.mjs",
-    "serve:docs": "pnpm storybook",
+    "build": "rimraf dist && pnpm build:copy-assets && pnpm build:js && pnpm build:esbuild",
+    "build:js": "tsup && pnpm build:fix-extension",
+    "build:fix-extension": "mv dist/synapse-react-client.js dist/synapse-react-client.mjs",
```

Comment:
For some reason, vitest won't read these when they are `.js`, even though we have `"type": "module"' in both projects. Just rename them after bundling.